scalableminds/webknossos
# Hiding pricing and features link in iframe

This PR hides the `pricing` and `features` links in the navbar in iframes.
### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Set `enableFrontpage` in `appilcation.conf` to true.
- Open the iframe test page in `tools/` and make sure the dataset, that the iframe displays is set to public.

### Issues:
- fixes #3986

------
